server/chat-commands/info.ts

```diff
@@ -630,6 +630,7 @@ export const commands: Chat.ChatCommands = {
 					};
 					details["Weight"] = `${pokemon.weighthg / 10} kg <em>(${weighthit} BP)</em>`;
 					const gmaxMove = pokemon.canGigantamax || dex.species.get(pokemon.changesFrom).canGigantamax;
+					const preEvolution = dex.species.get(newTarget.name).prevo;
```

Comment:
Is there a reason you didn't just do `pokemon.prevo`?

Reply:
No, there was no particular reason. `pokemon.prevo` is correct, thx